apps/api/src/domain/models/song/commands/add-song.command.ts

```diff
@@ -14,10 +15,12 @@ export class AddSong implements ICommand {
     @IsUUID()
     readonly id: string;
 
+    @NonEmptyString({ isOptional: true })
     @IsOptional()
```

Comment:
I realized after we talked that it will be nice to have these decorators simplified to one decorator with options for readability